packages/lesswrong/server/callbacks/votingCallbacks.ts

```diff
@@ -232,7 +233,7 @@ async function maybeCreateReviewMarket({newDocument, vote}: VoteDocTuple, collec
   if (post.postedAt.getFullYear() < (new Date()).getFullYear() - 1) return; // only make markets for posts that haven't had a chance to be reviewed
   if (post.manifoldReviewMarketId) return;
 
-  const annualReviewLink = 'https://www.lesswrong.com/tag/lesswrong-review'
+  const annualReviewLink = tagGetUrl({slug: 'lesswrong-review'}, {}, true)
```

Comment:
This link needs to be absolute because it's sent to Manifold's API

Reply:
Already is, third argument